docs/conf.py

```diff
@@ -10,15 +10,16 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
 
+import django
 
-# TODO: Use with autodoc
-# from django.conf import settings
+sys.path.insert(0, os.path.abspath(".."))
```

Comment:
Could use a comment on why we're inserting both this folder and the one above it to the path